conda-recipe/meta.yaml

```diff
@@ -18,7 +18,6 @@ requirements:
   run:
     - python {{PY_VER}}*,>=3
     - ophyd >=1.0.0,<1.1
```

Comment:
Perhaps this is a good opportunity to unlock this requirement to be `>=1.0.0,<2` until we actually see a problem with a `1.x` release